meta-luneos/recipes-devtools/simple-shader/simple-shader.bb

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Christophe Chapuis <chris.chapuis@gmail.com>
+
+DESCRIPTION = "Library of glib utilities."
+LICENSE = "Apache-2.0"
+SECTION = "devtools"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+
+DEPENDS = "virtual/egl"
+
+SRC_URI = "git://github.com/Tofee/simple-shader.git;protocol=https;branch=main"
+S = "${WORKDIR}/git"
+
+PV = "0.0.1+git${SRCPV}"
+SRCREV = "280047d9f5f159072a20f49a6d777a9f8da039f4"
+
+do_compile() {
+    cd ${S}
+    make test-shader LDFLAGS="${LDFLAGS}" CFLAGS="${CFLAGS}"
```

Comment:
oe_runmake would respect PARALLEL_MAKE and EXTRA_OEMAKE (where you can pass LDFLAGS and CFLAGS).